recipe/meta.yaml

```
@@ -3,7 +3,7 @@
 {% if CLANG_VERSION is not defined %}
 {% set CLANG_VERSION = version %}
 {% set CL_VERSION = "19.16.27032.1" %}
-{% set VCVER = "" %}
+{% set VCVER = "14.3" %}
```

```suggestion
{% set VCVER = "14.1" %}
```
We're currently depending on vs2017 which would be 14.1.